index.html

```diff
@@ -5,25 +5,197 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>First Capstone Project</title>
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <script
+  src="https://kit.fontawesome.com/6c59d03160.js"
+  crossorigin="anonymous"
+></script>
+    <title>Capstone Project Home Pages</title>
 </head>
 <body>
-    <header>
-        <nav class="navbar">
-          <a href="#"><h2 class="nav-logo">Tadesse</h2></a>
-          <ul class="nav-menu">
-            <li class="nav-item"><a href="#portfolio" class="nav-link">Portfolio</a><hr class="text-divide"></li>
-            <li class="nav-item"><a href="#about" class="nav-link">About</a><hr class="text-divide"></li>
-            <li class="nav-item"><a href="#contact-form" class="nav-link">Contact</a><hr class="text-divide"></li>
-            <li class="nav-icon" ><i class="fa fa-envelope-o"></i></li>
-          </ul>
-          <div class="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-          </div>
-        </nav>
-      </header>
+  <header>
+    <!-- Nav-bar -->
+    <div class="container">
+      <div class="header_nav">
+        <a href="https://www.facebook.com/"
+          ><i class="fa-brands fa-facebook-f"></i
+        ></a>
+        <a href="https://twitter.com/"
+          ><i class="fa-brands fa-twitter"></i
+        ></a>
+        <a href="#">English</a>
+        <a href="#">My Pages</a>
+        <a href="#">Logout</a>
+      </div>
+    </div>
+    <div class="nav">
+      <div id="hamburger">
+        <div></div>
+      </div>
+    </div>
+      <div id="overlay">
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="#programs">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsers">Sponsors</a>
+          <a href="./about.html">About</a>
+          <a href="#">
+            <span class="cc-campaign">Robo-Campaign</span>
+          </a>
+      </div>
```

Comment:
- [ ] For good semantics tag and as mentioned in the previous comment, I suggest you wrap all this block in a `<nav id="overlay">` tag as the container instead of a `div`,  all the navigation options in a `ul` tag and each option link wrapped in a `li` tag, i.e.:
```
   <nav id="overlay">
      <ul class="...">
        <li>
            <a href="./index.html" class="home-link">Home</a>
         </li>
         <li>
            <a href="#programs">Program</a>
         </li>
         <li>
             <a href="#speakers">Speakers</a>
          </li>
         ...

       </ul>
    </nav>
```